### PR TITLE
[TASK] Migrate to github actions codestyle check

### PR DESCRIPTION
**Pull request by lukaszuznanski**
https://github.com/TYPO3-Continuous-Integration/TYPO3-CI-Codestyle

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,6 +37,14 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run TypoScript lint"
         uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+  codestyle-check:
+    name: "Codestyle check"
```
Comment (oliverklee) [inline .github/workflows/ci.yml:41]: Please let's rename this job (and the new action and the Composer action) to "PHP CS Fixer" (`ci:php:cs-fixer`) to make it more clear what tool this is about.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -37,6 +37,14 @@ jobs:
...
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v2
+      - name: "Run Copypaste Detector"
```
Comment (oliverklee) [inline .github/workflows/ci.yml:46]: This looks like a copy'n'paste error. ;-)

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -37,6 +37,14 @@ jobs:
         uses: actions/checkout@v2
       - name: "Run TypoScript lint"
         uses: TYPO3-Continuous-Integration/TYPO3-CI-Typoscript-Lint@v1
+  codestyle-check:
+    name: "Codestyle check"
+    runs-on: ubuntu-20.04
```
Comment (oliverklee) [inline .github/workflows/ci.yml:42]: This job should depend on the PHP linter. (If the PHP is invalid, it does not make sense to check for duplication.)

**Review by oliverklee — CHANGES_REQUESTED** [review]
This also needs rebasing.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (oliverklee) [reply .github/workflows/ci.yml:41]: I've done one part in #227.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (lukaszuznanski) [reply .github/workflows/ci.yml:46]: that's true :) Thanks!

**Review by oliverklee — APPROVED** [review]

**Pull request merged**